app/ante.go

```diff
@@ -39,7 +39,6 @@ func NewAnteHandler(
 		ante.NewValidateSigCountDecorator(ak),
 		// ante.NewDeductFeeDecorator(ak, bankKeeper),
```

Comment:
can we remove dead code here?

Reply:
I would prefer not to actually I need to review something, good catch

Reply:
I've made some changes here and will pass to our team to review.  This file was pretty badly out of spec, and the last time I'd touched it was the 46 upgrade.  Mustafa, this was a really, really good catch.

Reply:
short answer: for now, no.  I want that to stay exactly as it is and then we will refactor.